packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1154,10 +1210,31 @@ async function getGCStateFromSnapshot(
         }
         const gcState = await readAndParseBlob<IGarbageCollectionState>(blobId);
         assert(gcState !== undefined, 0x2ad /* "GC blob missing from snapshot" */);
+        // Update latest serialized GC blobs
+        if (trackGCBlobState) {
+            serializedGCBlobs.set(key, JSON.stringify(generateSortedGCState(gcState)));
```

Comment:
Why do this for individual blobs and not for the overall GC state?

Reply:
I was overthinking this issue, I don't think I need this bit of code, and I just need to convert the root state to JSON.

Reply:
That will make the code simpler. We don't need to track a map but just one blob. If and when we decide to break this down into multiple blobs, we should implement this based on what we decide then.